docs/components/modeler/desktop-modeler/start-instance.md

```diff
@@ -8,6 +8,14 @@ After you have [deployed your process to Camunda 8](./connect-to-camunda-8.md),
 
 ![start instance icon](./img/start-instance-icon.png)
 
-After the instance was started successfully, a corresponding message is displayed:
+1. Deploy the diagram to **Camunda 8 SaaS**. You can skip this step if you have already deployed the diagram.
+
+![start instance successful](./img/start-instance-step-1.png)
+
+2. Start the process the instance with optional variables.
+
+![start instance successful](./img/start-instance-step-2.png)
+
+3. After the instance was started successfully, a corresponding message is displayed.
```

Comment:
As this isn't really an "action", this can be removed as a step. I would just note it on a line by itself.

Reply:
Okay, will adjust!